nightfall-deployer/truffle-config.js

```diff
@@ -95,6 +95,20 @@ module.exports = {
       networkCheckTimeout: 1000000000,
     },
 
+    staging_edge: {
```

Comment:
done. I renamed network to `staging_edge` because it simplifies the why I load secret keys in AWS. All networks starting with `staging` use the same set of keys, and thus i don't need to add a specific case for `edge`. I hope its ok

Reply:
💯